airbyte-integrations/connectors/source-asana/source_asana/manifest.yaml

```diff
@@ -0,0 +1,2622 @@
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/sections_compact_stream"
+        parent_key: "gid"
+        partition_field: "section_gid"
+
+  sections_stream:
+    $ref: "#/definitions/base_stream"
+    name: "sections"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/sections_schema"
+    $parameters:
+      path: "/sections/{{ stream_slice['section_gid'] }}"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/sections_compact_partition_router"
+
+  tasks_stream:
+    $ref: "#/definitions/base_stream"
+    name: "tasks"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/tasks_schema"
+    $parameters:
+      path: "/projects/{{ stream_slice['project_id'] }}/tasks"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        use_cache: true
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/projects_partition_router"
+
+  tasks_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/tasks_stream"
+        parent_key: "gid"
+        partition_field: "task_gid"
+
+  stories_compact_stream:
+    $ref: "#/definitions/base_stream"
+    name: "stories_compact"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/stories_compact_schema"
+    $parameters:
+      path: "/tasks/{{ stream_slice['task_gid'] }}/stories"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        use_cache: true
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/tasks_partition_router"
+
+  stories_compact_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/stories_compact_stream"
+        parent_key: "gid"
+        partition_field: "story_gid"
+
+  stories_stream:
+    $ref: "#/definitions/base_stream"
+    name: "stories"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/stories_schema"
+    $parameters:
+      path: "/stories/{{ stream_slice['story_gid'] }}"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/stories_compact_partition_router"
+
+  workspaces_stream:
+    $ref: "#/definitions/base_stream"
+    name: "workspaces"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/workspaces_schema"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
```

Comment:
Missing request parameter: `opt_fields`. Please add it according to the original implementation.